client/signup/config/flows.js

```diff
@@ -226,8 +224,22 @@ const Flows = {
 	filterDestination,
 
 	defaultFlowName: 'main',
-
-	getFlow( flowName ) {
+	resumingFlow: false,
+
+	/**
+	 * Get certain flow from the flows configuration.
+	 *
+	 * The returned flow is modified according to several filters here.
+	 *
+	 * `currentStepName` is the current step in the signup flow. It is used
+	 * to determine if any AB variations should be assigned after it is completed.
+	 * Example use case: To determine if a new signup step should be part of the flow or not.
+	 *
+	 * @param {String} flowName The name of the flow to return
+	 * @param {String} currentStepName The current step. See description above
+	 * @returns {Object} A flow object
+	 */
+	getFlow( flowName, currentStepName = '' ) {
```

Comment:
Should we rename this method to make it more explicit that the returned flow might change as the user progresses through the steps? Like `getModifiedFlow` or something